src/lib/ipsec/aes_128_gcm.lua

```diff
@@ -92,8 +92,8 @@ function aes_128_gcm:new (spi, keymat, salt)
    -- Compute subkey (H)
    o.hash_subkey = ffi.new("uint8_t[?] __attribute__((aligned(16)))", 16)
    aes_128_block(o.hash_subkey, o.keymat)
-   o.gcm_data = ffi.new("gcm_data[1] __attribute__((aligned(16)))")
-   ASM.aes_keyexp_128_enc_avx(o.keymat, o.gcm_data[0].expanded_keys)
+   o.gcm_data = ffi.new("gcm_data __attribute__((aligned(16)))")
+   ASM.aes_keyexp_128_enc_avx(o.keymat, o.gcm_data)
```

Comment:
I originally boxed `gcm_data` because I thought this was the only way to translate `&gcm_data` from the C test code that ships with the Intel code (maybe it even was before we swapped out the ASM with DynASM code?). Anyways this works and is much simpler. Again, I am no longer referring to the `expanded_keys` field explicitly.